### PR TITLE
Add German strings

### LINKED ISSUE #35: Formatting petition page
@alicoding @secretrobotron 

It's very exciting to see this campaign come together. I've seen the mockup here: 

http://ali.ngrok.io/data-retention/

I have a few questions: 

-Can the petition text be below the introduction?
-Can it be in italic? Ie: 

++++

You should sign this petition because it's super important. 

Below is the petition text: 

<i>Dear target - please stop data retention. </i>

### PR DESCRIPTION
**Pull request by alicoding**
closes #35

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne R?

File: public/index.css [hunk 1]
```diff
@@ -260,6 +260,23 @@
...
+    font-size: 2rem;
+  }
+
+  @media screen and (min-width: 768px) {
+    .headerH1 {
+      font-size: 4.3rem;
+    }
+
+    .headerH3 {
+      font-size: 2rem;
```
Comment (ScottDowne) [inline public/index.css]: headerH3 is the same here as without the query.

File: public/index.css [hunk 2]
```diff
@@ -260,6 +260,23 @@
   #header {
     text-align: center;
   }
+  .headerH1 {
+    font-size: 3.3rem;
+  }
+
+  .headerH3 {
```
Comment (ScottDowne) [inline public/index.css]: I suggest we move any changes to the headers to instead modify the default h1 and h3 values defined at the top of the file. I'm not convinced we need to changed what they were though.

**Comment by ScottDowne** [comment]
R- due to changes made on the header font sizes. The copy itself was fine.

**Pull request merged**